src/nvme/util.h

```diff
@@ -9,8 +9,88 @@
 #ifndef _LIBNVME_UTIL_H
 #define _LIBNVME_UTIL_H
 
+#include <endian.h>
```

Comment:
I got rid of them completely (since we already have `ccan/endian` available). The only aber dabei is that libnvme then can't provide the `nvme_mmio_read32/64` helpers (at least not as static inlines). I'm not sure if that is a problem.

Below, I just moved it to the `test/register.c` example.